docs/faq.rst

```diff
@@ -192,6 +192,9 @@ compute nodes are isolated, or in PC/laptop enviroments if you are travelling.
 query for a template resource that has not been previously accessed.
 If you know what are the templates you are planning to use, you could
 prefetch them using the Python client.
+In addition to the output-spaces that you specify, fmriprep will use
+'MNI152NLin2009cAsym', 'MNI152NLin6Asym' and 'OASIS30ANTs' templates, that
```

Comment:
I'm fairly sure ``MNI152NLin6Asym`` is only required if aroma or cifti outputs are specified. 
```suggestion
In addition to the output-spaces that you specify, *fMRIPrep* will use
``MNI152NLin2009cAsym``, ``MNI152NLin6Asym`` and ``OASIS30ANTs`` templates, that
```